src/main/java/htsjdk/samtools/util/IOUtil.java

```diff
@@ -889,6 +901,45 @@ public static String readFully(final InputStream in) {
         }
     }
 
+
+    /**
+     * Returns an iterator over the lines in a text file. The underlying resources are automatically
+     * closed when the iterator hits the end of the input, or manually by calling close().
+     *
+     * @param p a {@link Path} that is to be read in as text
+     * @return an iterator over the lines in the file
+     */
+    public static IterableOnceIterator<String> readLines(final Path p) {
```

Comment:
Could be `Files.lines(Path)` used instead of creating a new utility method?

Reply:
done